build-tools/packages/build-cli/src/commands/generate/sourceEntrypoints.ts

```diff
@@ -0,0 +1,111 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import type { PackageJson } from "@fluidframework/build-tools";
+import { Flags } from "@oclif/core";
+import type { TsConfigJson } from "type-fest";
+import {
+	generateSourceEntrypoints,
+	optionDefaults,
+	readPackageJson,
+	readTsConfig,
+} from "../../library/commands/generateSourceEntrypoints.js";
+import { ApiTag, BaseCommand } from "../../library/index.js";
+// AB#8118 tracks removing the barrel files and importing directly from the submodules, including disabling this rule.
+// eslint-disable-next-line import/no-internal-modules
+import { type ExportData, mapExportPathsFromPackage } from "../../library/packageExports.js";
+import type { CommandLogger } from "../../logging.js";
+
+/**
+ * Generates source entrypoints for Fluid Framework APIs to support API levels (/alpha, /beta. etc.).
+ */
+export default class GenerateSourceEntrypointsCommand extends BaseCommand<
+	typeof GenerateSourceEntrypointsCommand
+> {
+	static readonly description =
+		`Generates TypeScript source files that roll up APIs into different entrypoint files. The entrypoints are determined based on the "exports" field in package.json."`;
```

Comment:
I think it would be nice to clarify that contents of entrypoints are determined by tags.
So perhaps extend with "and contents of entrypoints are detered by API tags."